web_widget_x2many_2d_matrix/static/src/xml/web_widget_x2many_2d_matrix.xml

```diff
@@ -16,7 +16,13 @@
                         <th><t t-esc="widget.get_y_axis_label(y)" /></th>
                         <td t-foreach="widget.get_x_axis_values()" t-as="x" t-att-class="'oe_list_field_cell' + (widget.is_numeric ? ' oe_number' : '')" t-att-data-x="x">
                             <span t-att-class="widget.get_xy_value_class()">
-                                <input class="edit" t-att-data-id="widget.get_xy_id(x, y)" t-att-value="widget.format_xy_value(widget.get_xy_value(x, y))" />
+                                <t t-set="editable" t-value="widget.get_xy_editability(x, y)"/>
```

Comment:
This is a JS expression, but you're right, doing this works: `t-att-disabled="widget.get_xy_editability(x, y) ? undefined : 'disabled'"`